test/ir_Mitsubishi_test.cpp

```diff
@@ -652,6 +652,26 @@ TEST(TestMitsubishiACClass, VaneMode) {
   EXPECT_EQ(kMitsubishiAcVaneAutoMove - 1, ac.getVane());
 }
 
+TEST(TestMitsubishiACClass, VaneMode) {
```

Comment:
A test with the same name already exists. e.g. `VaneMode`
You need to change it for your new test.
e.g.
```c++
TEST(TestMitsubishiACClass, VaneLeft) {
```

Reply:
Changed to VaneLeft